packages/vuetify/src/components/VExpansionPanel/VExpansionPanels.ts

```diff
@@ -24,10 +24,13 @@ export default BaseItemGroup.extend({
   props: {
     accordion: Boolean,
     disabled: Boolean,
+    flat: Boolean,
+    hover: Boolean,
```

Comment:
focusable already does this?

Reply:
I don't think so, no. Focusable goes a step further and applies a style to the header when expanded, which may or may not be desired. Hover only applies a this style on hover. If you think this prop is redundant with focusable, I'll remove it, but I would personally use hover along with flat if it were available.